configuration/viperConfig.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func LoadConfiguration() error {
-	logger := GetLogger("Viper")
+	logger := NewLogger("mongodb")
```

Comment:
Creating a new logger everywhere seems a bit overhead, contributing to increased scope complexity and making the code less readable. Instead, consider utilizing the official log library. Alternatively, if structured logging is your goal, the standard library offers a package named `slog` package that supports all the functionality you have inside logger.go, which simplifies the logging process. Another option is to leverage external libraries like logrus or zerolog for more advanced logging features.

Here's an example illustrating how this can be a bit counterintuitive: we have a logger named `mongodb` within the initialization of the Viper configuration.

Reply:
You are correct, I studied about slog that you sent to me, but I need to adjust a lot of files for this. Can I do this in another PR?

Reply:
Sure!